pytools/graph.py

```diff
@@ -232,7 +239,38 @@ def compute_topological_order(graph):
                 # so either this is a leaf or all children have been visited
                 visiting.remove(node)
                 visited.add(node)
-                reverse_order.append(node)
+
+                if key is None:
+                    reverse_order.append(node)
+                else:
+                    # {{{ move every 'u' from heap to reverse_order s.t. 'node'->'u'
```

Comment:
What is `u`? What does the `->` arrow mean?

Reply:
It was supposed to mean there exists an edge from `node` to `u` in the `graph`. Looks like it needs some clarification.

Reply:
Resolving as that part of code no longer exists.